app/utils/before_request.py

```diff
@@ -0,0 +1,41 @@
+import jwt
+
+from flask import request
+
+from models import Account
+from settings import TOKEN_TYPE, SECRET_KEY, TOKEN_ALGORITHM
+from utils.exceptions import EmptyBodyException, SerializerValidationError
+
+
+def authenticate() -> None:
+    """
+    Decorator for authenticate user(set current user to request)
+    """
+
+    if request.headers.get('Authorization') is None:
+        request.user = None    # type: ignore
```

Comment:
so that means I can just pass not Authorization header and use any endpoints?

Reply:
yes, you can use all endpoints how not authorized user but you may not have permissions to example delete update etc

Reply:
right, forgot that we have unauthorized functionality